docs/onvif-configuration.md

```diff
@@ -77,10 +81,14 @@ pod, you can update the Configuration like this:
 helm repo add akri-helm-charts https://deislabs.github.io/akri/
 helm install akri akri-helm-charts/akri-dev \
     --set useLatestContainers=true \
-    --set onvifVideo.enabled=true \
-    --set onvifVideo.capacity=2
+    --set onvif.enabled=true \
+    --set onvif.brokerPod.image.repository="ghcr.io/deislabs/akri/onvif-video-broker:latest-dev" \
+    --set onvif.capacity=2
 ```
 
+## Disabling automatic service creation
+By default, the generic ONVIF Configuration will create services for (all the brokers of) a specific Akri Instance and (all the brokers of) an Akri Configuration. Disable the create of Instance level services and Configuration level services by setting `--set onvif.createInstanceServices=false` and `--set onvif.createConfigurationService=false`, respectively.
```

Comment:
why is 'all of the brokers of' parenthetical?

Reply:
I guess I wanted to emphasize "Instance" level service vs "Configuration" level services. Will happily remove the parentheses.